09Ianuarie/src/ro/sci/ale/ian09/temperature/TemperatureTest.java

```diff
@@ -0,0 +1,22 @@
+package ro.sci.ale.ian09.temperature;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class TemperatureTest {
+
+    @Test
+    void fahrenheitToCelsiusTest() {
+        double result = Temperature.fahrenheitToCelsius(98.6);
+        assertEquals(37, result);
+    }
+
+    @Test
+    void fahrenheitToCelsiusEquals() {
+        double result = Temperature.fahrenheitToCelsius(98.6);
+        assertNotEquals(-37, result);
```

Comment:
these 2 tests test the same thing, because if result==37 then clearly result!=-37

Another test would be to convert some other fahrenheit value to celsius and see if that is valid